packages/pigeon/platform_tests/ios_unit_tests/ios/Runner/async_handlers.h

```diff
@@ -0,0 +1,27 @@
+// Autogenerated from Pigeon (v0.1.19), do not edit directly.
+// See also: https://pub.dev/packages/pigeon
+#import <Foundation/Foundation.h>
+@protocol FlutterBinaryMessenger;
+@class FlutterError;
+@class FlutterStandardTypedData;
```

Comment:
You can `#import <Flutter/Flutter.h>` instead of forward declaring here.

Reply:
That was intentional since forward declaration results in faster compilation and I wanted clients to opt into including flutter symbols into their files as much as possible.